rs/nns/governance/src/neuron/types.rs

```diff
@@ -935,7 +934,12 @@ impl Neuron {
     }
 
     /// Get the 'public' information associated with this neuron.
-    pub fn get_neuron_info(&self, now_seconds: u64, requester: PrincipalId) -> NeuronInfo {
+    pub fn get_neuron_info(
+        &self,
+        voting_power_economics: &VotingPowerEconomics,
+        now_seconds: u64,
+        requester: PrincipalId,
+    ) -> NeuronInfo {
```

Comment:
Adding a parameter here results in lots of "fallout" changes.

When a test is affected, the only thing I needed to do to make the test work as before is to pass `&VotingPowerEconomics::DEFAULT`.

When the caller is production code (i.e. not test code), `governance.voting_power_economics()` can be passed.